packages/discord.js/src/util/Sweepers.js

```diff
@@ -425,6 +425,9 @@ class Sweepers {
     let items = 0;
 
     for (const guild of this.client.guilds.cache.values()) {
+      // We may be unable to sweep the cache if the guild is unavailable and was never patched
+      if (!guild.available) continue;
```

Comment:
```suggestion
      if (guild.available) continue;
```

Is the goal here to check if the guild is available before continuing? Right now it's checking if the guild is not available before continuing. I'm assuming you'd want the guild to be available if you want it to continue with sweeping

Reply:
I think you've misunderstood what `continue` does - it stops executing this item in the loop skips to the next item / iterable.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/continue

Reply:
I stand corrected 👍🏻 

I'll go take a trip down Mozillas keywords list